Gemfile.lock

```diff
@@ -24,10 +24,12 @@ GEM
     ethon (0.11.0)
       ffi (>= 1.3.0)
     eventmachine (1.2.7)
+    eventmachine (1.2.7-x64-mingw32)
```

Comment:
Is there a specific reason why we added the dependency again with variant `x64-mingw32`? same for `ffi` and `nokogiri`

Reply:
I think it's specifically for windows installs, @zmabry2 you're working on a Windows machine right? `mingw32` is a GNU emulator.

Reply:
Yes, I am using a windows machine. I can do the owrk on my Macbook if that would resolve these types of issues

Reply:
I don't think it's that big an issue, I think the installation should skip those dependencies if they aren't supported for other platforms.